addons/editor-theme3/addon.json

```diff
@@ -324,9 +327,75 @@
   "enabledByDefault": false,
   "presets": [
     {
-      "name": "3.0 Tweaks",
+      "name": "High contrast",
+      "id": "contrast",
+      "description": "Scratch 3.0's upcoming block color palette shown in Scratch Lab",
+      "values": {
+        "motion-color": "#80B5FF",
+        "looks-color": "#CCB3FF",
+        "sounds-color": "#E19DE1",
+        "events-color": "#FFD966",
+        "control-color": "#FFBE4C",
+        "sensing-color": "#85C4E0",
+        "operators-color": "#7ECE7E",
+        "data-color": "#FFA54C",
+        "data-lists-color": "#FF9966",
+        "custom-color": "#FF99AA",
+        "Pen-color": "#13ECAF",
+        "sa-color": "#12EDD5",
+        "comment-color": "#FEF49C",
+        "input-color": "#FFFFFF",
+        "text": "black"
+      }
+    },
+    {
+      "name": "Dark",
+      "id": "new-dark",
+      "description": "Dark versions of the default colors that look good in dark themes",
+      "values": {
+        "motion-color": "#2461B7",
+        "looks-color": "#633EAC",
+        "sounds-color": "#863286",
+        "events-color": "#AE8500",
+        "control-color": "#B07700",
+        "sensing-color": "#468CAF",
+        "operators-color": "#438C43",
+        "data-color": "#B46313",
+        "data-lists-color": "#A84210",
+        "custom-color": "#973545",
+        "Pen-color": "#2A7468",
+        "sa-color": "#2A797C",
+        "comment-color": "#242422",
```

Comment:
I don't think this preset should make comments dark gray, especially if it's meant to be less dark than the other "dark" one.